### PR TITLE
feat(cli): add unique-cli read command for indexed chunk retrieval

### PR DESCRIPTION
**Pull request by aakunique**
## Summary
- Adds `unique-cli read <cont_id>` — retrieves all indexed text chunks for a known document via `Content.search()` with an id filter, no vector search or query string needed
- Updates the `unique-cli-search` SKILL.md with a `search` vs `read` decision table so agents know which command to use in each situation
- 12 new unit tests covering all edge cases (empty/None chunks, sort order, API errors, call shape)

## Changes

**`unique_sdk/cli/commands/read.py`** (new)
- `cmd_read(state, cont_id)` — validates `cont_*` prefix, calls `Content.search()`, sorts chunks by `order` (null-safe), formats plain text with page annotations
- Guards for None chunks, empty chunks (with "may still be ingesting" message), and `APIError`

**`unique_sdk/cli/cli.py`**
- `@main.command(name="read")` Click command; errors to stderr + exit code 1

**`unique_sdk/cli/shell.py`**
- `do_read()` REPL handler

**`unique_sdk/cli/skills/unique-cli-search/SKILL.md`**
- Added `search` vs `read` decision table at the top
- Added "Reading a Document by ID" section with example output and citation note

**`unique_sdk/tests/cli/test_read.py`** (new)
- 12 tests: bad ID format, not found, API error, empty/None chunks, sorted output, null order, header content, API call shape verification

## Test plan
- [x] `uv run pytest tests/cli/test_read.py -v` — 12/12 passed
- [x] `uv run ruff check` + `ruff format --check` — clean on all changed files
- [x] pre-commit hooks passed on commit

Made with [Cursor](https://cursor.com)

### CODE REVIEW

**Review by mfadler — APPROVED** [review]
Looks good to me.

File: unique_sdk/unique_sdk/cli/skills/unique-cli-search/SKILL.md [hunk 1]
```diff
@@ -6,14 +6,27 @@ description: >-
...
 
-# Unique CLI -- Knowledge Base Search
+# Unique CLI -- Knowledge Base Search & Document Read
+
+## `search` vs `read` — which command to use
+
+| Situation | Command |
+|-----------|---------|
+| You have a **query or topic** and want to find relevant chunks across documents | `unique-cli search "<query>"` |
+| You already have a **`cont_*` ID** and want the **full indexed text** of that document | `unique-cli read <cont_id>` |
```
Comment (andreasunique) [inline unique_sdk/unique_sdk/cli/skills/unique-cli-search/SKILL.md:25]: Better describe here what the full index text means, that it OCRD the document the Image describtions extracted from the pdfs so that the agent know ther is no more OCR needed in oder to read the document.

File: unique_sdk/unique_sdk/cli/shell.py [hunk 2]
```diff
@@ -394,6 +394,28 @@ def do_cite(self, arg: str) -> None:
             return
         self._print(cmd_cite_file(self.state, positional[0], pages))
 
+    def do_read(self, arg: str) -> None:
```
[the remark this answers is not in the document]
Comment (aakunique) [reply unique_sdk/unique_sdk/cli/shell.py:398]: Fixed in [bfc93f39e](https://github.com/Unique-AG/ai/commit/bfc93f39e) — added `read` to `OVERVIEW_HELP` and `docs/cli/commands.md`.

File: unique_sdk/unique_sdk/cli/commands/read.py [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    """Return all indexed text chunks for *cont_id* as plain text.
+
+    Args:
+        state: Shell state carrying user/company credentials.
+        cont_id: A content ID (``cont_...``) to retrieve.
+
+    Returns:
+        A formatted string of chunks, or an error message prefixed with
+        ``read:``.
+    """
```
[the remark this answers is not in the document]
Comment (aakunique) [reply unique_sdk/unique_sdk/cli/commands/read.py:30]: Fixed in [bfc93f39e](https://github.com/Unique-AG/ai/commit/bfc93f39e) — `read` now calls `state.is_content_within_workspace(cont_id)` before fetching chunks, using the same `.unique-search.json` workspace boundary as `rm`/`mv`. Three tests added.

File: unique_sdk/unique_sdk/cli/commands/read.py [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+        text = (chunk.get("text") or "").strip()
+        if not text:
+            continue
+        start = chunk.get("startPage")
+        end = chunk.get("endPage")
+        if start and end:
+            page_ref = f"[p.{start}]" if start == end else f"[p.{start}-{end}]"
+            lines.append(f"{page_ref} {text}")
+        else:
+            lines.append(text)
```
[the remark this answers is not in the document]
Comment (aakunique) [reply unique_sdk/unique_sdk/cli/commands/read.py:86]: Fixed in [dfaf203bb](https://github.com/Unique-AG/ai/commit/dfaf203bb) — page labels now use `is not None` checks and fall back to `startPage` when `endPage` is missing. Existing `startPage`+`endPage` pairs unchanged.

File: unique_sdk/unique_sdk/cli/skills/unique-cli-search/SKILL.md (hunk 1 above)
Comment (aakunique) [reply unique_sdk/unique_sdk/cli/skills/unique-cli-search/SKILL.md:25]: Addressed in [dfaf203bb](https://github.com/Unique-AG/ai/commit/dfaf203bb) — added a paragraph explaining that `read` returns pre-ingested text (OCR, PDF/Office extraction, image descriptions) so no further OCR or file parsing is needed.

File: unique_sdk/unique_sdk/cli/skills/unique-cli-search/SKILL.md [hunk 5]
```diff
@@ -6,14 +6,29 @@ description: >-
...
+
+**What "full indexed text" means:** the platform has already ingested the document — OCR for scanned pages, extracted text from PDFs and Office files, and image descriptions from figures/charts. `read` returns that pre-processed text directly; you do **not** need to download the file or run OCR yourself.
```
Comment (andreasunique) [inline unique_sdk/unique_sdk/cli/skills/unique-cli-search/SKILL.md:27]: If you still find the returned data to be empty you better download the file and check yourself that rarely some files could not have been ingested, and must be inspected directly
